spec/abci++/abci++_tmint_expected_behavior.md

```diff
@@ -15,20 +15,23 @@ synchronously and there will be no byzantine process. In these frequent, benign
 
 * Tendermint will decide in round 0;
 * `PrepareProposal` will be called exactly once at the proposer process of round 0, height _h_;
-* `ProcessProposal` will be called exactly once at all processes except the proposer of round 0, and
+* `ProcessProposal` will be called exactly once at all processes, and
   will return _accept_ in its `Response*`;
-* `ExtendVote` will be called exactly once at all processes
-* `VerifyVoteExtension` will be called _n-1_ times at each validator process, where _n_ is the number of validators; and
-* `FinalizeBlock` will be finally called at all processes at the end of height _h_, conveying the same prepared
-  block that all calls to `PrepareProposal` and `ProcessProposal` had previously reported for height _h_.
+* `ExtendVote` will be called exactly once at all processes;
+* `VerifyVoteExtension` will be called _n-1_ times at each validator process, where _n_ is the
```

Comment:
```suggestion
* `VerifyVoteExtension` will be called up to _n-1_ times at each validator process, where _n_ is the
```

Reply:
Here, were are describing the base case scenario, when network is behaving well, and processes don't fall behind. In this case, all votes will reach their destinations (everyone will be able to vote), so each node will need to verify _precisely n-1_ votes.

Having said this, I should probably clarify the scenario in the text... if you missed it means it's not clear enough